### ISSUE TITLE
Return focus to clicked element when closing with esc-key

### ISSUE THREAD

**Issue by daevid66** [isaacplmann/ngx-contextmenu#105]
Hi, thanks for really useful component. 

I'm trying to get my app accessible for people that are not using a mouse. I am able to open the menu by keyboard and navigate it with cursor keys, but when I close the menu with ESC-key the focus is lost, meaning that I need to tab through all the page again to get back to where I was before.

This could be solved either by resetting the focus to the element that triggered the menu.

A possible workaround is that I save the active object when the menu is opened and reset focus on the close event, but I cant get the close event to work as expected.

**Comment by isaacplmann** [comment]
Yes, I've been meaning to add this for a while.  I want to use the `@angular/cdk` [a11y](https://material.angular.io/cdk/a11y/api) utilities to handle this.  If you want to tackle this (more difficult) PR, I can coach you through it.
